CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for Crate Data JDBC Client
 Unreleased
 ==========
 
+ - Fix: Added assumeMinServerVersion=9.5 as default for crate-jdbc.
```

Comment:
It's actually not a fix, but an improvement. I would.

Changed the `assumeMinServerVersion` default to "9.5", because CrateDB emulates Postgres version 9.5. This change eliminates initial `SET SESSION` statements which are ignored in the server-side anyway.